_assets/scripts/tag_version.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+
+set -e
+set -o pipefail
```

Comment:
nit : These two set statements could be clubbed into 1 like this -> `set -euo pipefail`